core/components/css-utilities/Miscellaneous/Miscellaneous.story.tsx

```diff
@@ -24,20 +24,631 @@ export const miscellaneous = () => {
+                alt="Example"
+                className="object-fit-cover w-100 h-100 rounded-10 border border-primary"
+              />
+            </Card>
+            <Text appearance="subtle" size="small" className="text-center mt-2 color-primary-dark">
+              Covers entire area, maintaining aspect ratio
+            </Text>
+          </div>
+          <div className="w-25 mb-3">
+            <div className="text-center mb-2 color-accent3-dark font-weight-bold">object-fit-contain</div>
+            <Card className="vh-25 overflow-hidden shadow-s bg-secondary-lightest p-2">
+              <img
+                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjIwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KIDxnPgogIDxyZWN0IHdpZHRoPSIzMDAiIGhlaWdodD0iMjAwIiBmaWxsPSIjZGRkZGRkIi8+CiAgPHJlY3QgeD0iNTAiIHk9IjUwIiB3aWR0aD0iMjAwIiBoZWlnaHQ9IjEwMCIgZmlsbD0iIzU1NTU1NSIvPgogIDx0ZXh0IHg9IjUwJSIgeT0iNTAlIiBmb250LWZhbWlseT0iQXJpYWwiIGZvbnQtc2l6ZT0iMThweCIgZmlsbD0id2hpdGUiIHRleHQtYW5jaG9yPSJtaWRkbGUiIGRvbWluYW50LWJhc2VsaW5lPSJtaWRkbGUiPkV4YW1wbGUgSW1hZ2U8L3RleHQ+CiA8L2c+Cjwvc3ZnPg=="
+                alt="Example"
+                className="object-fit-contain w-100 h-100 rounded-10 border border-accent3"
+              />
+            </Card>
+            <Text appearance="subtle" size="small" className="text-center mt-2 color-accent3-dark">
+              Fits within the box without cropping
+            </Text>
+          </div>
+          <div className="w-25 mb-3">
+            <div className="text-center mb-2 color-success-dark font-weight-bold">object-fit-fill</div>
+            <Card className="vh-25 overflow-hidden shadow-s bg-secondary-lightest p-2">
+              <img
+                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjIwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KIDxnPgogIDxyZWN0IHdpZHRoPSIzMDAiIGhlaWdodD0iMjAwIiBmaWxsPSIjZGRkZGRkIi8+CiAgPHJlY3QgeD0iNTAiIHk9IjUwIiB3aWR0aD0iMjAwIiBoZWlnaHQ9IjEwMCIgZmlsbD0iIzU1NTU1NSIvPgogIDx0ZXh0IHg9IjUwJSIgeT0iNTAlIiBmb250LWZhbWlseT0iQXJpYWwiIGZvbnQtc2l6ZT0iMThweCIgZmlsbD0id2hpdGUiIHRleHQtYW5jaG9yPSJtaWRkbGUiIGRvbWluYW50LWJhc2VsaW5lPSJtaWRkbGUiPkV4YW1wbGUgSW1hZ2U8L3RleHQ+CiA8L2c+Cjwvc3ZnPg=="
+                alt="Example"
+                className="object-fit-fill w-100 h-100 rounded-10 border border-success"
+              />
+            </Card>
+            <Text appearance="subtle" size="small" className="text-center mt-2 color-success-dark">
```

Comment:
Please use some better example to clearly define the difference between these classes. Right now all of them looks similar 
<img width="868" alt="image" src="https://github.com/user-attachments/assets/0c5d4a74-ca9c-4089-abd7-2ee7b5139017" />